tests/test_defaults.py

```diff
@@ -109,6 +114,28 @@ def test():
             ('method', 'GET'), ('path', '/skip'), ('status', 200)
         )
 
+    def test_exception_counter_metric(self):
+        self.metrics(export_defaults=False)
+
+        @self.app.route('/error')
+        def test():
+            abort(501)
+
+        self.client.get('/error')
+
+        self.assertMetric(
+            'flask_http_request_exceptions_total', '1.0',
+            ('method', 'GET'), ('path', '/error'), ('status', 501)
+        )
+
```

Comment:
These asserts somehow do not want to pass - am I missing something here, @rycus86 ?
As they fail across 4 tests, perhaps I didn't declare/init the related metric proplery?

Reply:
Drop the `export_defaults=False` from the setup on line 118 - this is a new default metric, so you do want the defaults exported.

Reply:
:facepalm: fixed that, but the regexp still won't match. :thinking: Judging from what I can see from the test output, there's no value for the metric present.

Reply:
Maybe try raising an exception instead of `abort`, maybe that's handled internally by Flask, so perhaps it doesn't go to that exception path? (mostly guessing here)

Reply:
Tried that, but that appears to be not handled by the test client at all (had to catch it explicitly when calling `.get()` in the test, otherwise it would've errored).
hm.

Reply:
Ah yes, I forgot that the test client is a bit different. Aren't there any other tests that raise exceptions?

Reply:
There are, at least using `abort()`. It's still odd. The only explanation I can come up with is that the exception only arrives there if the error was not handlled by an error handler? But the code doesn't look that way (but following that around in the source of flask was tricky, so I might have missed something).